lib/docusign_dtr/auth/error.rb

```diff
@@ -36,7 +36,7 @@ def general_error
         return DocusignDtr::InvalidGrant if error_code.match?(/grant/)
         return DocusignDtr::ApiLimitExceeded if error_code.match?(/HOURLY_APIINVOCATION_LIMIT_EXCEEDED/)
 
-        DocusignDtr::ConsentRequired.new(error_message)
+        DocusignDtr::ConsentRequired if error_code.match?(/consent_required/)
```

Comment:
so we don't want to return an error message anymore?

also, I think we should return some sort of default error in case `error_code` does not match any of the codes we're checking
we were using this `ConsentRequired` as the default, so if we do not want to continue doing that we could define a new one to be the default (StandardError maybe)

thoughts?

Reply:
I was thinking about that while doing other stuff
We can use error_message for standard errors